prime-angular-frontend/src/app/lib/modules/ngx-busy/ng-busy.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+
+export class NgBusyService {
+  private message: string;
+  private isShowSpinner: boolean;
+
+  constructor() { };
+
+  public setMessage(message: string): void {
```

Comment:
TS offers access to getters and setters 
https://www.typescripttutorial.net/typescript-tutorial/typescript-getters-setters/

Reply:
I used that earlier and decided not to afterwards for some reason. Now using default accessors